### DISCUSSION TITLE
[Feature request] Single wallet selection (filter)

### DISCUSSION THREAD

**Discussion by vildantursic** [WalletConnect/walletconnect-monorepo#1636] [Developer Support]
Is there a possibility to create feature _similar to v1_ where we can filter Wallets, but if we filter only 1 wallet it would directly initiate connection to that wallet **without prompting WalletConnect modal**?
Thank you in advance 🙏

**Comment by xzilja** [comment]
That's actually something we started working on or beta.5/beta.6, so you should be able to do it soonish 👍

**Reply by vildantursic** [reply]
Implemented it with beta.4 wagmi for now, but this is great guys, thank you!
